tests/operators/test_update.py

```diff
@@ -0,0 +1,139 @@
+"""Test for update in trajectory and density.
+
+Only finufft, cufinufft and gpunufft support update.
+"""
+
+import numpy as np
+import numpy.testing as npt
+import pytest
+from pytest_cases import parametrize_with_cases, parametrize, fixture
+from helpers import (
+    param_array_interface,
+    to_interface,
+    from_interface,
+)
+from mrinufft import get_operator
+from case_trajectories import CasesTrajectories
+
+
+@fixture(scope="module")
+@parametrize(
+    "n_batch, n_coils, n_trans, sense",
+    [
+        (1, 1, 1, False),
+        (3, 1, 1, False),
+        (1, 4, 1, True),
+        (1, 4, 1, False),
+        (1, 4, 2, True),
+        (1, 4, 2, False),
+        (3, 2, 1, True),
+        (3, 2, 1, False),
+        (3, 4, 2, True),
+        (3, 4, 2, False),
+    ],
+)
+@parametrize_with_cases(
+    "kspace_locs, shape",
+    cases=CasesTrajectories,
+    glob="*random*",
+)
+@parametrize(backend=["gpunufft"])
```

Comment:
@alineyyy you can add `finufft` and `cufinufft` here after you add similar support in it. (Remember to also update points for the grad_plan)